deploy/template.yaml

```diff
@@ -878,6 +879,10 @@ Resources:
             TableName: !Ref SessionsTable
         - SSMParameterReadPolicy:
             ParameterName: !Sub ${Environment}/core/self/*
+        - SSMParameterReadPolicy:
+            ParameterName: !Sub ${Environment}/core/credentialIssuers/*
+        - SSMParameterReadPolicy:
+            ParameterName: !Sub ${Environment}/core/credentialIssuers
```

Comment:
Are both these policies needed, the first doesn't cover the second?

Reply:
Ah yeah true. I just did a copy past of where we have it else where but its not needed here.